lib/upipe-blackmagic/upipe_blackmagic_source.cpp

```diff
@@ -545,6 +545,12 @@ HRESULT DeckLinkCaptureDelegate::VideoInputFrameArrived(
             else if (upipe_bmd_src->tff)
                 uref_pic_set_tff(uref);
 
+            IDeckLinkTimecode *timecode;
+            if (VideoFrame->GetTimecode(bmdTimecodeRP188Any, &timecode) == S_OK) {
+                uint32_t bcd[2] = { 1, timecode->GetBCD() };
```

Comment:
I discovered that the BCD from the Decklink API is byte-swapped compared to the ffmpeg side data.  I wonder if I should call a different function (GetComponents) or whether I can just byte-swap it.

Do we use any builtin functions anywhere?  I see some such as "expect" and "ctz" which are guarded by a gnuc check, and some "cpu_supports" guarded by other checks.

I think I'll just call the other function.